myuw/data/category_links_import.csv

```diff
@@ -1,31 +1,31 @@
+PageAcademics,Grades & Transcripts,all,,,https://sdb.admin.uw.edu/sisStudents/uwnetid/transcript.aspx,Order Official Transcripts,https://www.uwb.edu/registration/transcript,Order Official Transcripts,https://sdb.admin.uw.edu/sisStudents/uwnetid/transcript.aspx,Order Official Transcripts,yes
 PageAcademics,Grades & Transcripts,all,https://sdb.admin.uw.edu/students/uwnetid/unofficial.asp,Unofficial Transcript,,,,,,,yes
-PageAcademics,Graduation,all,,,http://www.washington.edu/students/reg/grad.html,Application for Graduation,https://www.uwb.edu/registration/graduation/apply,Applying for Graduation,http://www.tacoma.uw.edu/node/37921,Applying to Graduate,no
-PageAcademics,Tools & Software,all,,,http://scout.uw.edu/seattle,Scout: Discover the UW,http://scout.uw.edu/bothell,Scout: Discover the UW,http://scout.uw.edu/tacoma,Scout: Discover the UW,no
-PageAcademics,Tools & Software,all,http://www.washington.edu/itconnect/wares/uware/tableau-software/,Tableau Data Analysis Software,,,,,,,no
+PageAcademics,Graduation,all,,,https://registrar.washington.edu/enrollment-and-records/graduation-commencement-and-diplomas/,Application for Graduation,https://www.uwb.edu/registration/graduation/apply,Applying for Graduation,http://www.tacoma.uw.edu/node/37921,Applying to Graduate,no
+PageAcademics,Tools & Software,all,,,https://scout.uw.edu/seattle,Scout: Discover the UW,https://scout.uw.edu/bothell,Scout: Discover the UW,https://scout.uw.edu/tacoma,Scout: Discover the UW,no
+PageAcademics,Tools & Software,all,https://itconnect.uw.edu/wares/uware/tableau-software/,Tableau Data Analysis Software,,,,,,,no
 PageAcademics,Tools & Software,all,https://panopto.uw.edu/,Panopto,,,,,,,yes
 PageAcademics,Tools & Software,all,https://itconnect.uw.edu/wares/,Software & hardware for the UW,,,,,,,yes
 PageCalendar,Other Calendars,all,,,http://www.washington.edu/students/reg/calendar.html,All Academic Calendars,http://www.uwb.edu/calendar/academic,Bothell Academic Calendar,http://www.tacoma.uw.edu/node/41353,Tacoma Academic Calendars,no
-PageCalendar,Other Calendars,all,,,http://www.washington.edu/calendar/,Campus Event Calendar,http://www.uwb.edu/calendar,Bothell Campus Event Calendar,http://www.tacoma.uw.edu/home/calendar,Tacoma Campus Event Calendar,no
-PageCalendar,Other Calendars,all,,,http://depts.washington.edu/thehub/hub-events/event-calendars/,HUB Event Calendar,http://www.bothell.washington.edu/sea/events,Student Engagement & Activities Events,,,yes
+PageCalendar,Other Calendars,all,,,https://www.washington.edu/calendar/,Campus Event Calendar,https://www.uwb.edu/calenda,Bothell Campus Event Calendar,https://www.tacoma.uw.edu/home/calendar,Tacoma Campus Event Calendar,no
+PageCalendar,Other Calendars,all,,,https://hub.washington.edu/hub-events/event-calendars/,HUB Event Calendar,https://www.uwb.edu/sea/events,Student Engagement & Activities Events,,,yes
```

Comment:
https://www.uwb.edu/calenda 
is missing an "r" and should be 
https://www.uwb.edu/calendar

Reply:
everything else looks good!